<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(64, 64, 95);</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>591</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Login</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>591</width>
        <height>371</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>download.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(124, 0, 0);
background-color: rgba(0, 43, 63, 0);
font: 14pt &quot;Terminal&quot;;

</string>
      </property>
      <property name="text">
       <string>Face recongition </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(124, 0, 0);
background-color: rgba(0, 43, 63, 0);
font: 14pt &quot;Terminal&quot;;

</string>
      </property>
      <property name="text">
       <string>Enter the password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="PASSWORD">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#PASSWORD{
	rgb(0, 0, 57)
	padding:10px;
	font: 12pt &quot;Terminal&quot;;
	color: rgb(131, 0, 0);
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="LOGIN">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CLOSE">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>210</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QLabel" name="FAILWORD">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(124, 0, 0);
background-color: rgba(0, 43, 63, 0);
font: 14pt &quot;Terminal&quot;;

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Mainform</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>581</width>
        <height>381</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>8 - mainform.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="train_user">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>Train User</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reports_tab2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>220</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>Reports</string>
      </property>
     </widget>
     <widget class="QPushButton" name="attendt_entry">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>Attendence entry</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LOGOUT">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>290</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Training</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>581</width>
        <height>381</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>8 - training.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>321</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(124, 0, 0);
background-color: rgba(0, 43, 63, 0);
font: 14pt &quot;Terminal&quot;;

</string>
      </property>
      <property name="text">
       <string>Training process</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="personame_training">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#PASSWORD{
	rgb(0, 0, 57)
	padding:10px;
	font: 12pt &quot;Terminal&quot;;
	color: rgb(131, 0, 0);
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="person_name">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(124, 0, 0);
background-color: rgba(0, 43, 63, 0);
font: 14pt &quot;Terminal&quot;;

</string>
      </property>
      <property name="text">
       <string>Enter person name</string>
      </property>
     </widget>
     <widget class="QLabel" name="person_name_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(124, 0, 0);
background-color: rgba(0, 43, 63, 0);
font: 14pt &quot;Terminal&quot;;

</string>
      </property>
      <property name="text">
       <string>training capture counts</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="training_capture_counts">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#PASSWORD{
	rgb(0, 0, 57)
	padding:10px;
	font: 12pt &quot;Terminal&quot;;
	color: rgb(131, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
     <widget class="QPushButton" name="train">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>train</string>
      </property>
     </widget>
     <widget class="QPushButton" name="goback_training">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>290</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>goback</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Face Recongition</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>581</width>
        <height>381</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>8 - facereco.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Record">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>Record</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>321</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(124, 0, 0);
background-color: rgba(0, 43, 63, 0);
font: 14pt &quot;Terminal&quot;;

</string>
      </property>
      <property name="text">
       <string>ATTENDANCE ENTRY</string>
      </property>
     </widget>
     <widget class="QPushButton" name="face_recong_back">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>230</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>GO BACK </string>
      </property>
     </widget>
     <widget class="QLabel" name="wating_attendance">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(124, 0, 0);
background-color: rgba(0, 43, 63, 0);
font: 14pt &quot;Terminal&quot;;

</string>
      </property>
      <property name="text">
       <string>waiting.......</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Reports</string>
     </attribute>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>581</width>
        <height>381</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>8 - reports.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>321</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(124, 0, 0);
background-color: rgba(0, 43, 63, 0);
font: 14pt &quot;Terminal&quot;;

</string>
      </property>
      <property name="text">
       <string>ATTENDANCE REPORTS</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>80</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(124, 0, 0);
background-color: rgba(0, 43, 63, 0);
font: 8pt &quot;Terminal&quot;;

</string>
      </property>
      <property name="text">
       <string>Select Date</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>80</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="REPORTTAB4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>130</y>
        <width>191</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="reports_back">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>310</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>GO BACK </string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>681</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>49</y>
      <width>31</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>50</y>
      <width>61</width>
      <height>381</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>419</y>
      <width>631</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
